src/models/accountModel.ts

```diff
@@ -0,0 +1,58 @@
+import dbConnection from "../dbConnection/dbConnection";
+
+export interface Account {
+    customer_id: string,
+    account_type: string,
+    balance: number,
+    interest_rate: number,
+    custom_label: string,
+    account_number: number
+}
+
+export const addMoney = (amount: number, account_number: number): Promise<any> => {
+    const query = `UPDATE accounts SET balance =  (balance + ?)
+                                                  WHERE account_number = ?`
+
+    return new Promise((resolve, reject): void => {
```

Comment:
Having looked into it, I'm pretty sure you can just use `await` on the query (inside an async function) rather than having to return a promise like this